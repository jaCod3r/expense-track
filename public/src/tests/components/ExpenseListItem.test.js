import React from 'react';
import { shallow } from 'enzyme';
import { expenses } from '../redux/fixtures/expenses.js';
import ExpenseListItem from '../../components/ExpenseListItem.jsx';

test('should render expense list with expense', () => {
  const wrapper = shallow(<ExpenseListItem {...expenses[0]} />)
  expect(wrapper).toMatchSnapshot()
})