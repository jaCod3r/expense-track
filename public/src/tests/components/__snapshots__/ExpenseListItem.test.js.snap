// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list with expense 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h4>
      Description: 
      Gum
    </h4>
  </Link>
  <p>
    Amount: 
    $2.00
  </p>
  <p>
    Created At: 
    December 31st 1969
  </p>
</div>
`;
